doc/api/next_api_changes/removals/26852-OG.rst

```diff
@@ -3,3 +3,10 @@
 
 ... of the `.dates` module are removed without replacements. These were
 undocumented and not exported.
+
+Julian dates in Matplotlib were calculated from a Julian date epoch: ``jdate =
+(date - np.datetime64(EPOCH)) / np.timedelta64(1, 'D')``.  Conversely, a Julian
+date was converted to datetime as ``date = np.timedelta64(int(jdate * 24 *
+3600), 's') + np.datetime64(EPOCH)``. Matplotlib was using
+``EPOCH='-4713-11-24T12:00'`` so that 2000-01-01 at 12:00 is 2_451_545.0 (see
```

Comment:
```suggestion
``EPOCH='-4713-11-24T12:00'`` so that 2000-01-01 at 12:00 is ``2_451_545.0`` (see
```
possibly to clarify that it is a "Python float".

But clearly an improvement to me.